measurements.py

```diff
@@ -34,19 +34,25 @@ def compute_low(data: AnalogSpan, mid: float):
 
 class VoltageMeasurer(AnalogMeasurer):
     peak_to_peak = Measure("Peak-to-Peak", type=float,
-                           description="Peak to Peak", units="V")
-    minimum = Measure("Min", type=float, description="Minimum", units="V")
-    maximum = Measure("Max", type=float, description="Maximum", units="V")
-    mean = Measure("Mean", type=float, description="Mean", units="V")
-    mid = Measure("Mid", type=float, description="Midpoint", units="V")
+                           description="Voltage difference between maximum and minimum points.", units="V")
+    minimum = Measure(
+        "Min", type=float, description="Lowest voltage value of the waveform.", units="V")
+    maximum = Measure(
+        "Max", type=float, description="Highest voltage value of the waveform.", units="V")
+    mean = Measure("Mean", type=float,
+                   description="Average voltage level over the waveform.", units="V")
+    mid = Measure("Mid", type=float,
+                  description="Midpoint voltage between maximum and minimum.", units="V")
 
     amplitude = Measure("Amplitude", type=float,
-                        description="Amplitude", units="V")
+                        description="Magnitude from base to top value.", units="V")
 
-    top = Measure("Top", type=float, description="High Level", units="V")
-    base = Measure("Base", type=float, description="Low Level", units="V")
+    top = Measure("Top", type=float,
+                  description="Most prevalent voltage of the upper 50% of the waveform.", units="V")
```

Comment:
Should this be "Most prevalent voltage above the midpoint" to use and reference the existing "mid" measure name?